File: Praktikum Pemrograman I Modul 3/Praktikum I Modul3 24-10-2023/Python/PRAK302-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# Input nilai
nilai = int(input())

# Pengkondisian
if nilai >= 80:
    print(f"A")
elif nilai >= 70 and nilai <= 79:
    print(f"B")
elif nilai >= 60 and nilai <= 69:
    print(f"C")
elif nilai >= 50 and nilai <= 59:
    print(f"D")
elif nilai < 50:
    print(f"E")

File: Praktikum Pemrograman I Modul 3/Praktikum I Modul3 24-10-2023/C/PRAK302-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // Deklarasi Variabel
    int nilai;

    // Input
    scanf("%d", &nilai);

    // Pengkondisian
    if (nilai >= 80){
        printf("A");
    }
    else if (nilai >= 70 && nilai <= 79){
        printf("B");
    }
    else if (nilai >= 60 && nilai <= 69){
        printf("C");
    } 
    else if (nilai >= 50 && nilai <= 59){
        printf("D");
    } 
    else if (nilai < 50) {
        printf("E");
    }

    return 0;
}